test/playback-watcher.test.js

```diff
@@ -933,64 +933,64 @@ QUnit.module('PlaybackWatcher download detection', {
 });
 
 loaderTypes.forEach(function(type) {
-  QUnit.test(`detects ${type} appends without buffer changes and excludes`, function(assert) {
-    this.setup();
-    const loader = this.mpc[`${type}SegmentLoader_`];
-    const track = {label: 'foobar', mode: 'showing'};
+  if (type !== 'subtitle') {
```

Comment:
perhaps we should just return in here if `type === 'subtitle'` would keep the changes down. LGTM otherwise.

Reply:
I tried that but it didn't seem to work.

Reply:
That seems strange. It wouldn't execute for other loader types of returning early on subtitles?